bot/constants.py

```diff
@@ -190,12 +190,14 @@ class Emojis:
     dice_5 = "<:dice_5:755891608091885627>"
     dice_6 = "<:dice_6:755891607680843838>"
 
-    issue = "<:IssueOpen:629695470327037963>"
-    issue_closed = "<:IssueClosed:629695470570307614>"
-    pull_request = "<:PROpen:629695470175780875>"
-    pull_request_closed = "<:PRClosed:629695470519713818>"
-    pull_request_draft = "<:PRDraft:829755345425399848>"
-    merge = "<:PRMerged:629695470570176522>"
+    # These icons are from Github's repo https://github.com/primer/octicons/
+    issue_open = "<:IssueOpen:852294683731165184>"
+    issue_closed = "<:IssueClosed:852294683723694140>"
+    issue_draft = "<:IssueDraft:852294683592884284>"  # Not currently used by Github, but here for future.
+    pull_request_open = "<:PROpen:629695470175780875>"
```

Comment:
This emoji, pull_request_open, should be reuploaded.
Current one is below:
![image](https://user-images.githubusercontent.com/71233171/121459570-8684ad80-c979-11eb-8664-2c97121341c6.png)

New one:
![image](https://user-images.githubusercontent.com/71233171/121459608-913f4280-c979-11eb-9e4b-17d56a55e8a3.png)


However, it doesn't seems like this is updated on the octicon repo yet